less/index.less

```
@@ -53,7 +53,157 @@ table {
     box-sizing: border-box;
 }
 
-html, body {
-    height: 100%;
+html {
+	height: 100%;
+	font-size: 62.5%
 }
 
+.center() {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.right-hand() {
+	display: flex;
+	justify-content: flex-end;
+}
+
+@resume-color: rgb(243, 235, 235);
+
+p {
+	font-size: 1.5rem;
+	padding: 15px;
+	width: 100%;
+	line-height: 1.5;
+	font-family: Arial, Helvetica, sans-serif
+}
+
+em {
+	font-style: italic;
+}
+
+strong {
+	font-weight: bold;
+}
+
+h1, h2, h3 {
+	font-size: 2.5rem;
+	padding: 5px 30px;
+	line-height: 1.5;
+	font-family: 'Patua One';
+}
+
+.container {
+	max-width: 800px;
+	margin: 0 auto;
+	width: 100%;
+
+  }
+
+nav {
+	width: 100%;
+
+	img {
+		display: block;
+		width: 30%;
+		height: 30%;
+		margin: 10px auto;
+	
+		
+		@media (min-width: 500px) {
+		display: none;
+		}
+	}
+
+	ul {
+		.right-hand;
+
+		@media (max-width: 500px) {
+			flex-direction: column;
+			.center;
+		}
+	
+		li {
+			padding: 20px;
+
+			@media (max-width: 500px) {
+				border-bottom: 1px solid grey;
+			}
+		
+			a {
+				color: black;
+				text-decoration: none;
+				font-size: 2rem;
+			}
+			a:hover {
+				color: gray;
+				text-decoration: underline;
+				font-size: 2rem;
+			}
+		}
+	
+	}
+
+}
+
```

Your project overall looks great. I just want to point out something I'd personally like to see from my team. When you're developing, if its helpful to you to have a lot of extra spaces, that's fine. But after you're done building, do me a favor and go back and tighten things up a bit. It just makes it easier for me (and anyone else) to read. Also, get into the semi-colon habit. 

```suggestion

nav {
	width: 100%;
	img {
		display: block;
		width: 30%;
		height: 30%;
		margin: 10px auto;
		@media (min-width: 500px) {
		  display: none;
		};
	};
	ul {
		.right-hand;
		@media (max-width: 500px) {
			flex-direction: column;
			.center;
		};
		li {
			padding: 20px;
			@media (max-width: 500px) {
				border-bottom: 1px solid grey;
			};
			a {
				color: black;
				text-decoration: none;
				font-size: 2rem;
			};
			a:hover {
				color: gray;
				text-decoration: underline;
				font-size: 2rem;
			};
		};
	};
};

```